nix/packages/moq.nix

```diff
@@ -0,0 +1,32 @@
+{ self, nixpkgs, flake-utils, ... }:
+flake-utils.lib.eachDefaultSystem (system:
+  let
+    pkgs = import nixpkgs {
+      inherit system;
+    };
+  in
+    with pkgs;
+    {
+      packages = rec {
+        moq-relay = rustPlatform.buildRustPackage rec {
+          pname = "moq-relay";
+          version = "0.6.7";
+
+          src = ../../.;
+
+          cargoHash = "sha256-kOAulF1OqR1VBKSy15RURJEk2ZpgZIFxPwrr03RbvPk=";
```

Comment:
This should load Cargo.lock directly instead, otherwise the hash will have to be updated whenever Cargo.lock changes

Reply:
https://github.com/englishm/moq-rs/blob/cfd556680b05c811d21e72a07953b05f64a1a4a6/package.nix#L27-L30